pkg/virtual/workspaces/registry/rest.go

```diff
@@ -545,37 +381,7 @@ func InternalListOptionsToSelectors(options *metainternal.ListOptions) (labels.S
 var _ = rest.Creater(&REST{})
 
 // Create creates a new workspace
```

Comment:
```suggestion
// Create creates a new workspace.
```

Reply:
Done in commit ab6cf45a76494764713e1bd66e27413c22c7f29f